### ISSUE TITLE
Qt5Network.dll version 5.10.1.0

### ISSUE THREAD

**Issue by Virgus** [nitroshare/nitroshare-desktop#270]
Hello I'm trying using the windows version 0.34 but it seems buggy as it works every now and then (mainly from Android to PC, I never succeded transferring from PC to Android).

I just found the (still old) beta 0.40 at the suggested repository:
https://ci.quickmediasolutions.com/job/nitroshare/

Unfortunately the installer package is missing the Qt5Network.dll and I cannot find v5.10.1.0 anywhere in the web. I tried later versions but they're not recognized correctly.

Does anybody have a working 0.40 desktop version for windows ?
Are there any forks with working binaries for windows (unfortunately I'm not skilled enough to update and compile the source code with all the bug-fixes) ?
Does anybody know if there is a comparable app that works the beautiful way that Nitroshare does ?

Thanks and have a nice day,
V.

**Comment by vincenzogianfelice** [comment]
**This project is dead.**

a long time ago I also ran into these problems ... I have tried all the various versions of this program but none of them work ... not to mention the problems between android / windows. This project has not been maintained for 3 years and will remain so.
I give you some advice: use SAMBA and you will find yourself marvelous (it is the one you currently use). At least they are native things, so you avoid installing third party programs.

Why install programs when linux already have native systems that allow file transfer???

Take care, my friend

**Comment by Virgus** [comment]
Thanks @vincenzogianfelice I've opted for SAMBA too, and have been trying using LanDrive on Android but it only works fine on rooted phones (and it takes some time to start/stop service). On non-rooted phones port 445 is not available and I tried using LanDrive with [Multi Port Forwarder ](https://www.verigio.com/products/multi-port-forwarder/) but it's not a so comfortable method. What do you suggest exactly if I want to share "on the fly" content between Windows and Android ?

Thanks again and take care you too.
V.

**Comment by vincenzogianfelice** [comment]
Yes ... unfortunately landroid requires root permissions to be used .... luckily you can act differently. You can connect (from your smartphone) to the samba server and copy / paste files from / to the server from there. So, I can only recommend the excellent ["File Manager"](https://play.google.com/store/apps/details?id=com.alphainventor.filemanager&hl=it&gl=US) app. From this application you can map many networks on your smartphone! (not just samba!) and you can browse it safely

Enjoy!
